__Tests__/website.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('../source/scripts/sum');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8422468)
```suggestion
const sum = require("../source/scripts/sum");
```